src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (invertedRobot.hasOwnProperty(robot[key])) {
+      return null;
+    }
+
+    invertedRobot[String(robot[key])] = key;
```

Comment:
The explicit conversion to 'String' is not necessary because object keys are always strings in JavaScript. If the value is not a string, it will be coerced to a string automatically.